tests/Feature/FileAdder/IntegrationTest.php

```diff
@@ -594,3 +596,22 @@
 
     expect($result->getStatusCode())->toEqual(200);
 });
+
+it('will throw and exception and not create a record in database if file cannot be added', function () {
+
+    $this->expectException(DiskCannotBeAccessed::class);
```

Comment:
Instead of using `expectException`, tack on `throws` at the end of the test (see Pest docs for more info).

Reply:
Done.